src/app/api/config/index.ts

```diff
@@ -43,6 +43,7 @@ const getConfigFile = async () => {
  * @param configKeys keys to check against the config file
  */
 export async function getDbFieldNamesConfigStatus({ dbConfigData }: { dbConfigData: T_CONFIG_DB_VARIABLE[] }) {
+  return {};
```

Comment:
This was a way to by pass the error so that Brian can work on it later as the issue wasn't apparent. 
Iv'e removed it as the issue was just an extra bracket on the return function.